.ci/travis-compile.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+if [[ $DOCKER_BUILDTYPE ]]; then
```

Comment:
Should this be `[[ -n $DOCKER_BUILDTYPE ]]` ? It seems to contain a string below

Reply:
-n is the default operation, it doesn't need to be specified in bash.